js/store.js

```diff
@@ -1,19 +1,34 @@
 const store = {
+  _websites: null,
+
   init() {
-    this.addListeners();
+    this._read();
+  },
+
+  async _read() {
+    const data = await browser.storage.local.get('websites');
+    this._websites = clone(data.websites);
```

Comment:
Why store it in `_websites` at all if you are going to make this block on a write?

Can you just read into `_websites` in `init` instead from the storage instead must making this read from the property?